public/app/plugins/datasource/elasticsearch/components/QueryEditor/AnnotationQueryEditor.tsx

```diff
@@ -19,79 +19,83 @@ export function ElasticsearchAnnotationsQueryEditor(props: Props) {
-          <EditorField label="Time End">
-            <Input
-              type="text"
-              value={annotation.timeEndField}
-              onChange={(e) => {
-                onAnnotationChange({
-                  ...annotation,
-                  timeEndField: e.currentTarget.value,
-                });
-              }}
-            />
-          </EditorField>
-          <EditorField label="Text">
-            <Input
-              type="text"
-              value={annotation.textField}
-              onChange={(e) => {
-                onAnnotationChange({
-                  ...annotation,
-                  textField: e.currentTarget.value,
-                });
-              }}
-            />
-          </EditorField>
-          <EditorField label="Tags">
-            <Input
-              type="text"
-              placeholder="tags"
-              value={annotation.tagsField}
-              onChange={(e) => {
-                onAnnotationChange({
-                  ...annotation,
-                  tagsField: e.currentTarget.value,
-                });
-              }}
-            />
-          </EditorField>
-        </EditorRow>
-      </div>
+        <div>
+          <h6>Field mappings</h6>
+          <EditorRow>
+            <EditorField label="Time">
+              <Input
+                type="text"
+                placeholder="@timestamp"
+                value={annotation.timeField}
+                onChange={(e) => {
+                  onAnnotationChange({
+                    ...annotation,
+                    timeField: e.currentTarget.value,
+                  });
+                }}
+              />
+            </EditorField>
+            <EditorField label="Time End">
+              <Input
+                type="text"
+                value={annotation.timeEndField}
+                onChange={(e) => {
+                  onAnnotationChange({
+                    ...annotation,
+                    timeEndField: e.currentTarget.value,
+                  });
+                }}
+              />
+            </EditorField>
+            <EditorField label="Text">
+              <Input
+                type="text"
+                value={annotation.textField}
+                onChange={(e) => {
+                  onAnnotationChange({
+                    ...annotation,
+                    textField: e.currentTarget.value,
+                  });
+                }}
+              />
+            </EditorField>
+            <EditorField label="Tags">
+              <Input
+                type="text"
+                placeholder="tags"
+                value={annotation.tagsField}
+                onChange={(e) => {
+                  onAnnotationChange({
+                    ...annotation,
+                    tagsField: e.currentTarget.value,
+                  });
+                }}
+              />
+            </EditorField>
+          </EditorRow>
+        </div>
+
+        <div> </div>
```

Comment:
```suggestion
      {/*Empty div to preserve the bottom margin */}
      <div />
```
This could use a comment with an explanation.